bin/node/cli/src/service.rs

```diff
@@ -100,12 +104,16 @@ macro_rules! new_full_start {
 						keystore: builder.keystore(),
 						babe_config: sc_consensus_babe::BabeLink::config(babe_link).clone(),
 						shared_epoch_changes: sc_consensus_babe::BabeLink::epoch_changes(babe_link).clone()
+					},
+					grandpa: node_rpc::GrandpaDeps {
+						shared_voter_state: Arc::clone(&shared_voter_state),
+						shared_authority_set: shared_authority_set.clone(),
 					}
 				};
 				Ok(node_rpc::create_full(deps))
 			})?;
 
-		(builder, import_setup, inherent_data_providers)
+		(builder, import_setup, inherent_data_providers, shared_voter_state)
```

Comment:
Do you mind instead creating an `let rpc_setup = Some((shared_voter_state));` and returning that instead? Similar to what we do for the `import_setup`. It's futile right now as we only return one thing from the RPC setup but will make it easier if we need to add anything else in the future.